packages/host/src/QueryErrorHandler.tsx

```diff
@@ -11,9 +15,10 @@ export const QueryErrorHandler = () => {
   const [errorMessage, setErrorMessage] = useState<string | null>(null);
   const location = useLocation();
   const generalError = t('error.general-query-error');
+  const [authError] = useLocalStorage('/auth/error');
 
   useEffect(() => {
-    if (!!errorMessage) {
+    if (!!errorMessage && authError !== AuthError.Unauthenticated) {
```

Comment:
The `Unauthenticated` error is handled with a Modal, so there's no need to toast